test/unified_format.py

```diff
@@ -1131,16 +1205,14 @@ def run_scenario(self, spec):
 
         # process operations
         self.run_operations(spec['operations'])
+        self.verify_outcome(spec.get('outcome', []))
```

Comment:
Is this change important for some reason?

Reply:
No, I will revert it.